src/main/java/Main.java

```diff
@@ -0,0 +1,52 @@
+import db.Storage;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import model.FruitTransaction;
+import service.CsvFileReaderService;
+import service.CsvFileWriterService;
+import service.FruitService;
+import service.FruitTransactionParser;
+import service.impl.CsvFileReaderServiceImpl;
+import service.impl.CsvFileWriterServiceImpl;
+import service.impl.FruitServiceImpl;
+import service.impl.FruitTransactionParserImpl;
+import strategy.OperationHandler;
+import strategy.handlers.BalanceHandler;
+import strategy.handlers.PurchaseHandler;
+import strategy.handlers.ReturnHandler;
+import strategy.handlers.SupplyHandler;
+
+public class Main {
+    public static void main(String[] args) {
+        CsvFileReaderService readerService = new CsvFileReaderServiceImpl();

```

Comment:
It's better to make services independent and inject dependencies through the constructor. Also, to follow the SOLID principles, define an interface for the `CsvFileReaderService` and use it here.